examples/kind-cluster-configuration.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Cluster
```

Comment:
nit: we should probably move this under `hack` ?  as `examples` should probably contain CR examples only

Reply:
I think we propose to use this file in the documentation for testing the example.
`hack` folder is more for contributor.

Reply:
okay